src/models/SearchResultsBib.ts

```diff
@@ -0,0 +1,81 @@
+import type { SearchResult } from "../types/searchTypes"
+import type { ElectronicResource } from "../types/bibTypes"
+
+export default class SearchResultsBib {
+  id: string
+  title: string
+  yearPublished?: string
+  materialType?: string
+  publicationStatement?: string
+  eResources?: ElectronicResource[]
+  numPhysicalItems: number
+
+  constructor(result: SearchResult) {
+    this.id = result["@id"] ? result["@id"].substring(4) : ""
+    this.title = this.getTitleFromResult(result)
+    this.yearPublished = this.getYearFromResult(result)
+    this.materialType = result.materialType[0]?.prefLabel || null
+    this.publicationStatement = result.publicationStatement?.length
+      ? result.publicationStatement[0]
+      : null
+    this.eResources = result.electronicResources || null
+    this.numPhysicalItems = result.numItemsTotal || 0
```

Comment:
same idea as [this comment](https://github.com/NYPL/research-catalog/pull/17/files#r1313098518). lets double down on what these item counts are and mean.